Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/AddOns/SceneExporter/o3de_utils.py

```diff
@@ -24,22 +26,69 @@ def LookatEngineManifest():
             o3de_projects = data['projects']
             # Send if o3de is not installed
             engine_is_installed = True
+            o3de_projects = extend_project_list(o3de_projects)
             return o3de_projects, engine_is_installed
     except:
         o3de_projects = ['']
         engine_is_installed = False
         return o3de_projects, engine_is_installed
 
-def build_projects_list():
+def load_saved_projects():
+    """!
+    This function will load the users project.json list
+    """
+    # Find the installed modules path on users system
+    for addon_path in addon_utils.modules():
+        if addon_path.bl_info['name'] == 'O3DE_DCCSI_BLENDER_SCENE_EXPORTER':
+            addon_full_path = addon_path.__file__
+            # Lets just get the directory path
+            bpy.types.Scene.plugin_directory = Path(addon_full_path).parent
+    project_json_path = Path(bpy.types.Scene.plugin_directory, "projects.json")
+    if project_json_path.exists():
+        with open(Path(bpy.types.Scene.plugin_directory, "projects.json"), "r") as data_file:
+            data = json.load(data_file)
+            return data
+
+def extend_project_list(o3de_projects):
     """!
-    This function will build the o3de project list
+    This function will load the users project.json list and extend it to the engine manifest projects
     """
-    o3de_projects, engine_is_installed = LookatEngineManifest()
+    saved_o3de_projects = load_saved_projects()
+    # Check to see if these projects are already on list
+    if saved_o3de_projects not in o3de_projects:
+        o3de_projects.extend(saved_o3de_projects)
+    return o3de_projects
 
+def build_projects_list():
+    """!
+    This function will check to see if O3DE is installed, looks at the o3de engine manifest projects
+    and the user saved project.json, builds the o3de project list
+    """
+    o3de_projects, engine_is_installed = look_at_engine_manifest()
+    # Project list
     list_o3de_projects = []
-    
+    # Check to see if O3DE is installed.
     if engine_is_installed:
         # Make a list of projects to select from
         for project_full_path in o3de_projects:
-            list_o3de_projects.append((project_full_path, Path(project_full_path).name, project_full_path))
-    return list_o3de_projects
\ No newline at end of file
+            # Check to see if the project name might be 1 level up in this path if ending with project.
+            # For example we could have a path like this C:/Users/USERNAME/O3DE/Projects/loft-arch-vis-sample/Project
+            if Path(project_full_path).name == 'Project':
+                p = Path(project_full_path)

```

Comment:
style: generally avoid using single letter variable names

Reply:
Yeah.. that's a good tip, I am generally good at not doing this, I must have missed it. Done